src/main/asciidoc/_chapters/hbase-default.adoc

```diff
@@ -2207,3 +2207,39 @@ The percent of region server RPC threads failed to abort RS.
 .Default
 `-1`
 
+
+[[hbase.regionserver.slowlog.ringbuffer.size]]
+*`hbase.regionserver.slowlog.ringbuffer.size`*::
++
+.Description
+
+      Default size of ringbuffer to be maintained by each RegionServer in order
+      to store online slowlog responses. This is an in-memory ring buffer of
+      requests that were judged to be too slow in addition to the responseTooSlow
+      logging. The in-memory representation would be complete.
+      For more details: https://issues.apache.org/jira/browse/HBASE-22978
```

Comment:
Instead of pointing off to a JIRA, how about refer to your new "Get Slow Response Log from shell" section in the book?